src/reifyhealth/specmonstah/spec_gen.cljc

```diff
@@ -60,10 +60,14 @@
 (defn spec-gen-merge-overwrites
   "Finally, merge any overwrites specified in the schema or query"
   [{:keys [data] :as db} ent-name ent-attr-key]
-  (let [{:keys [spec-gen]} (sm/ent-schema db ent-name)]
-    (merge (lat/attr data ent-name ent-attr-key)
-           spec-gen
-           (ent-attr-key (sm/query-opts db ent-name)))))
+  (let [{:keys [spec-gen]} (sm/ent-schema db ent-name)
+        spec-generated-val (lat/attr data ent-name ent-attr-key)
+        query-opts         (ent-attr-key (sm/query-opts db ent-name))]
+    (cond-> (lat/attr data ent-name ent-attr-key)
```

Comment:
You bind the result of `(lat/attr data ent-name ent-attr-key)` to `spec-generated-val` in the let binding, but looks like it is never used. Should either remove the let binding or replace the expression in `cond->` with `spec-generated-val`. Personally, I think using the bound value is most readable since it is explicit about what it is.